src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/ParseRequestServiceTest.java

```diff
@@ -85,6 +91,16 @@ void getRoleAssignmentByActorAndCaseId_shouldThrowBadRequestWhenActorAndCaseIdIs
         );
     }
 
+    @Test
+    void getRoleAssignmentByActorAndCaseId_shouldThrowBadRequestWhenCaseIdIsNotValid() {
```

Comment:
Please rename method.

Reply:
Changed to getRoleAssignmentThrow400ForInvalidCaseId